client_cli/src/main.rs

```diff
@@ -683,13 +689,21 @@ mod asset {
         /// List assets
         #[clap(subcommand)]
         List(List),
+        /// Put a key-value in a store Asset defintion
+        SetStore(SetStore),
+        /// Remove a key-value via key in a store Asset definition
+        RemoveStore(RemoveStore),
+        /// Get a value from store Asset definition via key
+        GetStore(GetStore),
+        /// Update a value from store Asset definition via key
+        UpdateStore(UpdateStore),
```

Comment:
I think the following naming would reflect the API of Iroha more precise:

- `set-key-value` (with underlying `SetKeyValue` ISI)
- `get-key-value` (with underlying `FindAssetKeyValueByIdAndKey` query)
- `remove-key-value` (with underlying `RemoveKeyValue` ISI)